sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/Models/Confidence.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.SelfHelp.Models
+{
+    /// <summary> Confidence of the search result. </summary>
+    public readonly partial struct Confidence : IEquatable<Confidence>
```

Comment:
The name is too generic. Consider appending RP prefix or renaming it.
- use `prepend-rp-prefix` to automatically append PR name. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L52-L57).
- use `rename-mapping` to rename a model. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L58-L76)

Reply:
Thanks for your suggestion, but I am struggling to understand why we need to rename. This code is generated from ARM swagger and ARM was okay with our naming convention. Is the current naming convention breaking any build?

Reply:
> @archerzz Thanks for your suggestion, but I am struggling to understand why we need to rename. This code is generated from ARM swagger and ARM was okay with our naming convention. Is the current naming convention breaking any build?

@sahilpwr That's the requirement from .NET SDK arch board. As you may know, .NET SDK is not adopting garbage-in/garbage-out strategy, so there are quite many guidelines on the generated SDK.

Reply:
@archerzz updated the names